extensions/theme-defaults/themes/dark_vs.json

```
@@ -308,7 +312,7 @@
 				"support.type.property-name",
 				"variable.css",
 				"variable.scss",
-				"variable.other.less",
+				"source.css.less variable.other",
```

I beliebe we can now merge the 3 lines with:
```suggestion
				"variable.css",
				"variable.scss",
				"source.css.less variable.other",
```

---

I created https://github.com/microsoft/vscode/pull/197657 for that change